AdSecGHTests/Components/FlattenRebarGhTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿using AdSecCore.Builders;
+using AdSecCore.Helpers;
+
+using AdSecGH.Components;
+using AdSecGH.Parameters;
+
+using AdSecGHTests.Helpers;
+
+using Grasshopper.Documentation;
+using Grasshopper.Kernel;
+using Grasshopper.Kernel.Types;
+
+using Oasys.AdSec;
+using Oasys.AdSec.DesignCode;
+using Oasys.GH.Helpers;
+
+using Rhino.Geometry;
+
+using Xunit;
+
+namespace AdSecGHTests.Components {
+  [Collection("GrasshopperFixture collection")]
+  public class FlattenRebarGhTests {
+    private readonly FlattenRebarGhComponent component;
+    private readonly IDesignCode designCode = IS456.Edition_2000;
+    private bool valueChanged;
+
+    public FlattenRebarGhTests() {
+      component = new FlattenRebarGhComponent();
+    }
+
+    private ISection Section => new SectionBuilder().WithWidth(40).CreateSquareSection().Build();
```

Comment:
properties before constructor pls

Reply:
I had a different rule on my settings. I made it a field though